languages/java/exercises/concept/case-statement/src/test/java/PlayAnalyzerTest.java

```diff
@@ -0,0 +1,58 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+public class PlayAnalyzerTest {
+
+    @Test(expected = IllegalArgumentException.class)
+    public void test_exception(){
+        PlayAnalyzer.analyseOnField(13);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void test_exception_negative_number(){
+        PlayAnalyzer.analyseOnField(-1);
+    }
+
+    @Test
+    public void test_goal(){
+        Assert.assertEquals("goalie", PlayAnalyzer.analyseOnField(1));
```

Comment:
It looks like the other exercises tests use a fluent syntax for the assertions: https://github.com/exercism/v3/blob/master/languages/java/exercises/concept/basics/src/test/java/LasagnaTest.java#L16 Could you use the same style?